Makefile

```diff
@@ -42,14 +41,6 @@ hypershift-operator:
 
 .PHONY: control-plane-operator
 control-plane-operator:
-	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
```

Comment:
Whoops, this line is collateral damage. Will wait for e2e to finish before invalidating it with a new commit.

/hold

Reply:
/hold cancel